.github/awstest.yml

```diff
@@ -0,0 +1,29 @@
+name: Arcadia-Science/metagenomics AWS tests
+# This workflow can be triggered manually with the GitHub actions workflow dispatch button.
+# It runs the -profile 'test' on AWS batch
+
+on:
+  workflow_dispatch:
+jobs:
+  run-tower:
+    name: Run AWS tests
+    if: github.repository == 'Arcadia-Science/metagenomics'
+    runs-on: ubuntu-latest
+    steps:
+      # Launch workflow using Tower CLI tool action
+      - name: Launch workflow via tower
+        uses: nf-core/tower-action@v3
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}}
+          compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
+          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/metagenomics/work-${{ github.sha }}
+          parameters: |
+            {
+              "outdir": "s3://${{ secrets.AWS_S3_BUCKET }}/metagenomics/results-${{ github.sha }}"
+            }
+          profiles: test_illumina,test_nanopore,aws_tower
```

Comment:
Oooh can profiles be combined like this? would this submit two different runs to tower, one for illumina and one for nanopore, or is it just one? it would be really cool if `test_illumina` output a different multiqc than `test_nanopore`, but I'm guessing it doesnt.

Reply:
@mertcelebi do you think we could do this for the tower full data sets over at seqqc?

Reply:
Great question! We should test, I am not sure TBH. ie I am not sure how the combined profiles work.

Reply:
I'm not sure either, other than I know I would like to test the separate profiles. And yes they do output different multiqc reports, I've already tested that configuration!

Reply:
for now I've changed so it just has `test_illumina` although I would like to test both in the future, but starting simple for now